source/security.txt

```diff
@@ -13,13 +13,163 @@ Secure Your Data
 .. facet::
    :name: genre
    :values: reference
- 
+
 .. meta::
+   :keywords: ldap, authorize, ecs, aws, authenticate
    :description: Learn how to use {+driver-short+} to secure your data.
 
 .. toctree::
    :titlesonly:
    :maxdepth: 1
 
    /security/authentication
-   /security/enterprise-authentication
\ No newline at end of file
+   /security/enterprise-authentication
+
+Overview
+--------
+
+MongoDB supports multiple mechanisms that you can use to authenticate your application.
+This page contains code examples that show each of these mechanisms.
+
+.. tip::
+  
+  To learn more about any of the authentication mechanisms on this page, see the
+  :ref:`cpp-auth` and :ref:`cpp-enterprise-auth` pages.
+
+To use an authentication example from this page, copy the code example into the
+:ref:`sample application <cpp-auth-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as ``<hostname>``, with
+the relevant values for your MongoDB deployment.
+
+.. _cpp-auth-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/connect-sample-app.cpp
+      :language: cpp
+      :copyable: true
+      :linenos:
+      :emphasize-lines: 13-15
+
+SCRAM-SHA-256
+-------------
+
+.. literalinclude:: /includes/authentication.cpp
+   :language: cpp
+   :copyable: true
+   :start-after: // start-scram-sha-256
+   :end-before: // end-scram-sha-256
+
+To learn more about SCRAM-SHA-256 authentication, see :ref:`cpp-scram-sha-256` in
+the Authentication guide.
+
+SCRAM-SHA-1
+-----------
+
+.. literalinclude:: /includes/authentication.cpp
+   :language: cpp
+   :copyable: true
+   :start-after: // start-scram-sha-1
+   :end-before: // end-scram-sha-1
+
+To learn more about SCRAM-SHA-1 authentication, see :ref:`cpp-scram-sha-1` in
+the Authentication guide.
+
+MONGODB X.509
+-------------
+
+.. literalinclude:: /includes/authentication.cpp
+   :language: cpp
+   :copyable: true
+   :start-after: // start-x509
+   :end-before: // end-x509
+
+To learn more about X.509 authentication, see :ref:`cpp-x509` in
+the Authentication guide.
+
+MONGODB-AWS
+-----------
+
+The following sections show how to connect to MongoDB by using the MONGODB-AWS
+authentication mechanism. Each section shows how to create the connection URI
+for the specified method of retrieving AWS credentials.
```

Comment:
S: this intro should describe the difference between the methods (URI, env variables, etc)

Reply:
Added some description from the funds page about the methods